src/producer.cpp

```diff
@@ -44,6 +47,54 @@ ndn::Name IceflowProducer::prepareDataName(uint32_t partitionNumber) {
   return ndn::Name(m_pubTopic).appendNumber(partitionNumber);
 }
 
+void IceflowProducer::setupBackchannel() {
+  auto backChannelPrefixName = ndn::Name(m_nodePrefix);
+  backChannelPrefixName.append("backchannel");
```

Comment:
AFAIK, the nodePrefix is set once per IceFlow instance, not per edge.

Does using this prefix still work if there is more than one outgoing edge? It seems to me that this would cause multiple subscriptions under the same `backChannelPrefixName` to be made.

Reply:
Yeah, you are right, it might be better to replace this with an extension of the topic. I briefly tried that out before opening the PR but there was of course some kind of issue again, therefore I wanted to open this one as a basis that at least works as intended :sweat_smile: I try out the other approach again :)